gtest_parallel.py

```diff
@@ -1029,6 +1032,10 @@ def default_options_parser():
   parser.add_option('--serialize_test_cases', action='store_true',
                     default=False, help='Do not run tests from the same test '
                                         'case in parallel.')
+  parser.add_option('--log_to_stdout',
+                    choices=['yes', 'no'],
+                    default='yes',
+                    help='Print individual test output to stdout')
```

Comment:
Do we need the choices, or should this just be enabled by passing `--log_to_stdout`?

Reply:
In scrum discussion we decided to make it enabled by default. I'm not sure how to allow an option to turn it off without the choices.

Reply:
Ah ok, fair point. This is fine then.

Reply:
Oh, that would work too. Maybe `--suppress_individual_test_output`, as there is other output which would not be turned off

Reply:
Any of the suggested options are fine with me. Your call.